### PR TITLE
use kustomize for deployment

### PR DESCRIPTION
**Pull request by yhwang**
- update deployment tool to `kustomize`
- add `username` and `password` inputs to create
  an user entry in AppID instance for initial
  login
- disable other identification profiders in AppID instance, i.e.
  google, facebook for security concern

fix: #3 
Signed-off-by: Yihong Wang <yh.wang@ibm.com>

### CODE REVIEW

**Comment by yhwang** [comment]
verified both classic and vpc-gen2 on ibm cloud. going to merge it

**Comment by chinhuang007** [comment]
Sounds good. I am refreshing my workspace to use the latest and will update the final status once finish verification.

File: terraform/iks-classic/README.md [hunk 1]
```diff
@@ -49,12 +57,12 @@ to modify the playbook to accommodate your own environment.
...
 1. From the IBM Cloud menu select [Schematics](https://cloud.ibm.com/schematics/overview).
-   - Click **Create workspace**.   
-   - Enter a name for your workspace.   
+   - Click **Create workspace**.
+   - Enter a name for your workspace.
    - Click **Create** to create your workspace.
 2. On the workspace **Settings** page, enter the URL of this terraform
-   template: `https://github.com/IBM/auto-kubeflow/tree/main/terraform/iks-classic`.
-   - Select the Terraform version: Terraform 0.14.
+   template: `https://github.com/IBM/auto-kubeflow/tree/main/terraform/iks_classic`.
```
Comment (chinhuang007) [inline terraform/iks-classic/README.md:64]: There is a typo, should be dash, not underscore

File: terraform/iks-classic/README.md [hunk 2]
```diff
@@ -24,6 +30,8 @@ to modify the playbook to accommodate your own environment.
 |  kube_version          | kubernetes version | string  |  | 1.20.7 |  |
 |  appid_plan            | The plan for AppID | string  |  | lite |  |
 |  appid_name            | Instnace name for AppID service | string  |   | appid-instance |  |
+|  username              | The user name to login to kubeflow dashboard | string  | ✓ |  |  |
+|  password              | The password to login to kubeflow dashboard | string  | ✓ |  |  |
```
Comment (chinhuang007) [inline terraform/iks-classic/README.md:34]: Consider suggesting password to be sensitive.

**Pull request merged**